rules/engine.go

```diff
@@ -293,11 +293,8 @@ func (e *baseEngine) addRule(rule DynamicRule,
 
 func (e *v3Engine) Run() {
 	e.logger.Info("Rules engine options", zap.Object("options", &e.options), zap.Int("rules", len(e.ruleMgr.rules)))
-	prefixSlice := []string{}
-	prefixes := e.ruleMgr.prefixes
 	// This is a map; used to ensure there are no duplicates
-	for prefix := range prefixes {
-		prefixSlice = append(prefixSlice, prefix)
+	for prefix := range e.ruleMgr.watcherPrefixes {
```

Comment:
I would rather have a method for access to this, so all of the internals is kept within the rule mgr.